reviser/dir.go

```diff
@@ -58,7 +78,10 @@ func (d *SourceDir) walk(options ...SourceFileOption) fs.WalkDirFunc {
 		if !d.isRecursive && dirEntry.IsDir() && filepath.Base(d.dir) != dirEntry.Name() {
 			return filepath.SkipDir
 		}
-		if isGoFile(path) && !dirEntry.IsDir() {
+		if dirEntry.IsDir() && d.isExcluded(path) {
```

Comment:
move it before `if !d.isRecursive && dirEntry.IsDir() && filepath.Base(d.dir) != dirEntry.Name() {` (line77)